packages/business-rules-lib/src/validators/contact.validators.js

```diff
@@ -154,15 +191,33 @@ export const overseasPostcodeRegex = /^([a-zA-Z0-9 ]{1,12})$/i
  * @returns {Joi.StringSchema}
  */
 export const createUKPostcodeValidator = joi =>
-  joi.string().trim().min(1).max(12).required().pattern(ukPostcodeRegex).replace(ukPostcodeRegex, '$1 $2').uppercase().example('AB12 3CD')
+  checkCopyPasteValidator(joi, forbiddenCharsNumbersRegex)
+    .string()
+    .allowable()
+    .trim()
+    .min(1)
+    .max(maxPostcode)
+    .required()
+    .pattern(ukPostcodeRegex)
+    .replace(ukPostcodeRegex, '$1 $2')
+    .uppercase()
+    .example('AB12 3CD')
 
 /**
  * Create a validator to check/format overseas postcodes
  * @param {Joi.Root} joi the joi validator used by the consuming project
  * @returns {Joi.StringSchema}
  */
 export const createOverseasPostcodeValidator = joi =>
-  joi.string().trim().min(1).max(12).uppercase().required().pattern(overseasPostcodeRegex)
+  checkCopyPasteValidator(joi, forbiddenCharsNumbersRegex)
+    .string()
```

Comment:
Do we need to include `string()` on these if `checkCopyPasteValidator` is already extending the default string validator? Same question goes for the other uses of it.